### PR TITLE
chore(curriculum): update lecture video ID's to correct videos

### PR DESCRIPTION
**Pull request by moT01**
Updates the video ID for all challenges in each of these blocks:

- Browsing the Web Effectively
- Common Design Tools
- CSS Specificity, the Cascade Algorithm, and Inheritance
- Importance of Semantic HTML
- Styling Lists and Links
- User Centered Design
- User Interface Design Fundamentals
- What is CSS
- Working with Backgrounds and Borders
- Working with Forms
- Working with HTML Tools
- Working with Relative and Absolute Units
- Working with Tables
- Working with CSS Fonts
- Working with Flexbox
- Best Practices for Styling Forms

The videos ID's should match what's on the spreadsheet. There's a link to this PR on there by all the blocks that this updates.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lecture-styling-lists-and-links/672b952d3a5c603908841971.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 id: 672b952d3a5c603908841971
 title: Why Are Default Link Styles Important for Usability on the Web?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: 9ta8m4T9KYc
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lecture-styling-lists-and-links/672b952d3a5c603908841971.md:5]: This is the video id for `What is the CSS filter property`

**Review by Dario-DC — CHANGES_REQUESTED** [review]
It seems that the link for `Why Are Default Link Styles Important for Usability on the Web?` is wrong in the sheet.

File: curriculum/challenges/english/25-front-end-development/lecture-styling-lists-and-links/672b952d3a5c603908841971.md (hunk 1 above)
Comment (jdwilkin4) [reply curriculum/challenges/english/25-front-end-development/lecture-styling-lists-and-links/672b952d3a5c603908841971.md:5]: I just messaged Beau about updating the link with the correct one. He is just coming back from a conference so he will probably respond back within a day or so.

Comment (jdwilkin4) [reply curriculum/challenges/english/25-front-end-development/lecture-styling-lists-and-links/672b952d3a5c603908841971.md:5]: Beau updated the sheet.
Here is the correct link

https://www.youtube.com/watch?v=Jp3Qd4LU93E

**Review by Dario-DC — APPROVED** [review]

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**